projects/opendr_ws/src/perception/scripts/video_activity_recognition.py

```diff
@@ -101,23 +120,27 @@ def callback(self, data):
         :param data: input message
         :type data: sensor_msgs.msg.Image
         """
-        image = self.bridge.from_ros_image(data)
+        image = self.bridge.from_ros_image(data, encoding="rgb8")
         if image is None:
             return
 
-        x = self.preprocess(image.numpy())
+        x = self.preprocess(image.convert("channel_first", "rgb"))
```

Comment:
There's a typo in the argument producing an exception. 
`ValueError: format not in ('channels_first', 'channels_last')` originating from [here](https://github.com/opendr-eu/opendr/blob/develop/src/opendr/engine/data.py#L347).
```suggestion
        x = self.preprocess(image.convert("channels_first", "rgb"))
```